kymatio/scattering2d/frontend/torch_frontend.py

```diff
@@ -0,0 +1,181 @@
+__all__ = ['Scattering2D']
+
+import torch
+from ..backend import cdgmm, Modulus, SubsampleFourier, fft, Pad, unpad, BACKEND_NAME
+from ..filter_bank import filter_bank
+from ..utils import compute_padding
+from ..scattering2d import scattering2d
+from ...frontend.torch_frontend import Scattering
+
+if BACKEND_NAME != 'torch' and BACKEND_NAME != 'skcuda':
+    raise (RuntimeError('The only supported backend by the torch frontend are torch and skcuda.'))
```

Comment:
This line might lower code coverage, because I don't think it can ever be reached (I think the check in `..backend` will always raise before this raises)